webpack.config.js

```diff
@@ -10,7 +10,7 @@ const { spawn } = require('child_process');
 
 module.exports = {
   // original: entry: SRC_DIR + '/index.js'
-  entry: `${SRC_DIR}/main/index.js`,
+  entry: `${SRC_DIR}/public/main.js`,
```

Comment:
why the entry point change ?  it should be index.js since it is the react entry point for the bundler. Lets chat in person for more clarity.

Reply:
Yes you are correct. I did not need to change the entry point.